### ISSUE TITLE
Feature request: 2D scroll

### ISSUE THREAD

**Issue by lukas-buergi** [Seelge/TrackballScroll#3]
On Linux I'm able to set up my trackball for 2D scroll which works just like the 1D scroll this program does, only it can scroll horizontally too.

Is this possible with the api you are using for this?

Edit: Sorry for not looking into it myself, I have never gotten around to learing how to compile _anything_ on Windows... at some point I will, can't be that hard.

**Comment by Seelge** [comment]
Thanks for the suggestion, v1.1.0 includes horizontal scrolling and is available for download now.

If you wish to compile yourself on windows, it's basically opening the project and clicking build. Visual Studio Community 2015 is free but takes a while to install.

**Comment by lukas-buergi** [comment]
Thanks a  lot. :-)

It never turned out to be that easy when I tried it, but probably a relatively small program like this is a better place to start than what I previously tried. I guess I'll give it another try.

**Comment by Seelge** [comment]
> On Linux I'm able to set up my trackball for 2D scroll

Can you point me to a guide how to configure Linux to do this?

**Comment by lukas-buergi** [comment]
I use this as a startup script: https://gist.github.com/TT44bb/02186dd9d9533adee3f4dd691d3e5078
It doesn't quite work like I want it to and I don't know why. The 2D scrolling part works, the button remapping so I can also middle click doesn't. The line which is likely the problem is marked in the comments.

There are likely better options, but as soon as I've got a mostly working version motivation tends to drop... I just spent quite some time trying to get it right (not for the first time) and it just won't.

Edit: Oh and documentation is mostly in the "man" pages for the commands used... "xev" is useful for debugging, it displays raw input events. Also for the vmware problem you've got.
